locales/zh_TW/json.json

```diff
@@ -64,7 +64,7 @@
     "Antarctica": "南極洲",
     "Antigua And Barbuda": "安提瓜和巴布達",
     "Antigua and Barbuda": "安提瓜和巴布達",
-    "API": "應用程序接口",
+    "API": "API",
```

Comment:
In this case, you must also put this value in the [_excludes.json](https://github.com/Laravel-Lang/lang/blob/main/locales/zh_TW/_excludes.json) file so that the translation statistics generator does not react to it as not translated.

Reply:
got it, thanks!